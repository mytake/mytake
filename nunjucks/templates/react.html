{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
