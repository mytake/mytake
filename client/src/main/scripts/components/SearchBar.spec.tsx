import * as React from "react";
import * as renderer from "react-test-renderer";
import SearchBar from "./SearchBar";

jest.mock("./DropDown", () => ({
  default: "DropDown"
}));

test("SearchBar", () => {
  const tree = renderer
    .create(<SearchBar searchTerm={"search term"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
