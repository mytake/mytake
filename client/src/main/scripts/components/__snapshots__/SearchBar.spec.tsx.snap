// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar 1`] = `
<form
  action="javascript:void(0)"
  className="searchbar__form searchbar__form--"
  onSubmit={[Function]}
>
  <div
    className="searchbar__input-container searchbar__input-container--"
  >
    <input
      className="searchbar__input searchbar__input--"
      onChange={[Function]}
      results={5}
      type="search"
      value="search term"
    />
    <span
      className="searchbar__cancel searchbar__cancel--"
      onClick={[Function]}
    >
      <svg
        fill="none"
        height={24}
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="12"
          cy="12"
          r="10"
        />
        <line
          x1="15"
          x2="9"
          y1="9"
          y2="15"
        />
        <line
          x1="9"
          x2="15"
          y1="9"
          y2="15"
        />
      </svg>
    </span>
  </div>
  <button
    className="searchbar__button searchbar__button--"
    type="submit"
  >
    <svg
      fill="none"
      height={24}
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="11"
        cy="11"
        r="8"
      />
      <line
        x1="21"
        x2="16.65"
        y1="21"
        y2="16.65"
      />
    </svg>
  </button>
</form>
`;
